src/hammer-vlsi/defaults.yml

```diff
@@ -434,11 +434,16 @@ par:
   # If power_straps_mode is 'generate', which method to use.
   # Currently, the valid options are:
   # - by_tracks - Specify the power strap plan per layer in terms of tracks
+  # - by_tracks_hier - Only valid for hierarchical blocks- uses the top-level by_tracks to determine parameters
```

Comment:
I thought this was going in a separate PR?

Reply:
Oversight, will fix